static_frame/core/store.py

```diff
@@ -218,6 +234,12 @@ def __init__(self,
                 if (config.label_encoder != self._default.label_encoder or
                         config.label_decoder != self._default.label_decoder):
                     raise ErrorInitStoreConfig(f'config {label} has encoder/decoder inconsistent with default; align values and/or pass a default StoreConfig.')
+                if (config.read_max_workers != self._default.read_max_workers or
+                        config.read_chunksize != self._default.read_chunksize):
+                    raise ErrorInitStoreConfig(f'config {label} has read_max_workers/chunksize inconsistent with default; align values and/or pass a default StoreConfig.')
+                if (config.write_max_workers != self._default.write_max_workers or
+                        config.write_chunksize != self._default.write_chunksize):
+                    raise ErrorInitStoreConfig(f'config {label} has write_max_workers/chunksize inconsistent with default; align values and/or pass a default StoreConfig.')
```

Comment:
I wonder if we can consolidate all of these default checks into  a loop, where we just iterate over the attrs we require should be the same?